src/edtr-io/get-plugin-registry.tsx

```diff
@@ -104,12 +103,6 @@ export function getPluginRegistry(
       description: editorStrings.edtrIo.spoilerDesc,
       icon: createIcon(faCaretSquareDown),
     },
-    {
-      name: 'table',
-      title: editorStrings.edtrIo.table,
-      description: editorStrings.edtrIo.tableDesc,
-      icon: createIcon(faTable),
-    },
```

Comment:
If you want the old tables to still work for now, the plugin still has to be there. You could hide it, though, e.g. by removing/filtering it from the rows plugin registry.

Reply:
This file creates the registry. 
`table` is still part of the [`plugins.tsx`](https://github.com/serlo/frontend/blob/hide-table-plugin/src/edtr-io/plugins.tsx#L368) and [should still work](https://frontend-git-hide-table-plugin-serlo.vercel.app/entity/repository/add-revision/206397).

Reply:
I see, makes sense!